### ISSUE TITLE
can not use root as user in install

### ISSUE THREAD

**Issue by auburncodingclub** [Bercik1337/rt-auto-install#25]
You may wish to echo an alert for those (of us) silly enough to try to use root as a user (rather than / in addition to the echo about user privileges..).

Thank you for all of your hard work!!!!

**Comment by Bercik1337** [comment]
@auburncodingclub hey man. I've spent some time thinking about it, was about to make it today but... 1. it's not that easy 2. i'm not sure if it's all that good idea.

Whenever we use sudo, it changes our uid to 0, therefore programs see this as root user. (whoami and others). I was planning to edit CHECK_ROOT function or  SET_RTORRENT_USER but in both cases, we MUST see uid=0, that's what's this all about.

And even IF i could figure out a way to detect, if it's run as root or sudo, there's the problem with: it could cause problems for others.

For instance, some people use it in containers, and up there it's often root who runs all. This would make it very cumbersome for them to work around it.

So to sum it up - even i found a way to differenciate root vs sudo, I'm afraid it will cause more harm then good.

Only thing i can propose for now would be: changing 
`echo "${GREEN}Remember to run as root or with sudo.${NORMAL}"`
to something like
`echo "${GREEN}Remember to run with sudo and not as root user itself.${NORMAL}"`

Also modern distros usually setup user with sudo WITHOUT even setting up password for root user (like ubuntu).
